### DISCUSSION TITLE
gamepad - multiple mappings (advanced)

### DISCUSSION THREAD

**Discussion by Jopp-gh** [AntiMicroX/antimicrox#1077] [Q&A] [answered]
Is it possible to:

1. Press a button twice to launch a key and ignoring the simple press of the same button ?
2. Hold a button for 1-2  seconds to launch and hold a key, ignoring the simple press of the same button?

I checked the advanced settings but couldn't figure out how to do it.

Example (point 1): 
- A to jump (single press)
- Hold A for 1-2 seconds to crouch and stay crouched (sticky button), ignoring regular jump

Example (point 2):
– B to interact/use (game objects)
- press B twice to change weapon, ignoring simple B press

Are these advanced presses possible and if yes, are there some examples, please?

**Comment by pktiuk** [comment, marked as answer]
Unluckily no.   
AntiMicroX for now reacts only for presses of buttons (there are neither combinations: https://github.com/AntiMicroX/antimicrox/issues/298 nor reacting differently after some time)

**Reply by Jopp-gh** [reply]
Oh, I see, thanks.
Maybe you want to merge my request with issue 298.

**Reply by pktiuk** [reply]
If you want, you can prepare a Pull Request implementing this, but I would recommend firstly preparing Draft Pull Request to show general way you want to implement this issue. https://github.com/AntiMicroX/antimicrox/blob/master/CONTRIBUTING.md
